sycl/plugins/unified_runtime/ur/adapters/cuda/context.cpp

```diff
@@ -0,0 +1,154 @@
+//===--------- context.cpp - CUDA Adapter ----------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===-----------------------------------------------------------------===//
+
+#include "context.hpp"
+
+#include <cassert>
+
+/// Create a UR CUDA context.
+///
+/// By default creates a scoped context and keeps the last active CUDA context
+/// on top of the CUDA context stack.
+/// With the __SYCL_PI_CONTEXT_PROPERTIES_CUDA_PRIMARY key/id and a value of
+/// PI_TRUE creates a primary CUDA context and activates it on the CUDA context
+/// stack.
+///
+UR_APIEXPORT ur_result_t UR_APICALL
+urContextCreate(uint32_t DeviceCount, const ur_device_handle_t *phDevices,
```

Comment:
Convention in PI is to name variables in Pascal. Don't know if later post-merge builds would flag and fail because of this. @smaslov-intel @againull  : do you know?

Reply:
The original PI cuda plugin had a mixture of different styles so it seems like it hasn't been an issue previously